config.cfg.template

```diff
@@ -193,6 +193,20 @@ config = {
         # you'll want to set this to True to disable manual adding of packages
         "auto_tracking": False,
 
+        # authentication settings
+        "auth": {
+            # Regular expression used to match user identity (OpenID
+            # identifier, KRB5 principal etc.) to user name.  Identity
+            # must match this regex as a whole, user name is is
+            # content of group.
+            # Password-based auth or no auth
+            "user_re": "(.+)",
+            # OpenID
+            #"user_re": "http://(.+)\\.example\\.com/",
```

Comment:
Can you please put regex for Fedora openid here? either commented out or even make it the default